packages/@vuepress/markdown/lib/snippet.js

```diff
@@ -15,8 +15,10 @@ module.exports = function snippet (md, options = {}) {
       if (fs.existsSync(src)) {
         token.content = fs.readFileSync(src, 'utf8')
       } else {
-        token.content = 'Not found: ' + src
+        const errorMessage = `Not found: ${src}`
+        token.content = errorMessage
         token.info = ''
+        logger.error(errorMessage)
```

Comment:
I don't think temp variable is really useful here. Error log message can also be improved (now that it's also log in the server console)

Here is a suggestion
```
token.content = `Code snippet path not found: ${src}`
token.info = ''
logger.error(token.content)
```